backends/tofino/bf-asm/crash.cpp

```diff
@@ -257,7 +257,10 @@ static void crash_shutdown(int sig, siginfo_t *info, void *uctxt) {
         if (++threads_dumped < int(thread_ids.size())) {
             lock.unlock();
             pthread_exit(0);
-        } else { lock.unlock(); })
+        } else {
+            // Unlock.
```

Comment:
Why add the comment?

Reply:
Good catch! This is a clang-format issue. Because the statements are in a macro they are incorrectly formatted on the same line, which cpplint complains about. You either add `clang-format off`, `nolint`, or this comment to work around it.